src/transformers/models/t5/tokenization_t5.py

```diff
@@ -175,7 +195,9 @@ def __init__(
             legacy = True
 
         self.legacy = legacy
-        self.sp_model_kwargs = {} if sp_model_kwargs is None else sp_model_kwargs
+        self.sp_model = self.get_spm_processor()
+        self.vocab_file = vocab_file
+        self._extra_ids = extra_ids
```

Comment:
I see `self.vocab_file` and `self._extra_ids` are assigned twice here, is that expected?

Reply:
that a typo, good catch 😉